### DISCUSSION TITLE
2.6.0 release

### DISCUSSION THREAD

**Discussion by bigluds** [kingToolbox/WindTerm#1407] [General]
Dear kingtoolbox,

I am a fan of this tool and I would lire to know when you will release the 2.6.0 version of this great and promising tool.

Best regards

**Comment by FRomanMF** [comment]
I totally agree. I don't know your personal situation but it looks like development is abandoned. I think that this tool is awesome and I'm sure that some people could make donations to help continue development (myself included).
